### PR TITLE
Script to migrate label notifications to alerts

### PR DESCRIPTION
**Pull request by ambuznego**

### CODE REVIEW

File: lib/console/alerts/alert.ex [hunk 1]
```diff
@@ -34,7 +34,7 @@ defmodule Console.Alerts.Alert do
     |> cast(attrs, [:name, :organization_id, :last_triggered_at, :config, :node_type])
     |> validate_required([:organization_id, :config, :node_type])
     |> validate_required(:name, message: "Name cannot be blank")
-    |> validate_length(:name, max: 25, message: "Name cannot be longer than 25 characters")
+    |> validate_length(:name, max: 88, message: "Name cannot be longer than 88 characters") # TODO reduce to 25 after 2.0 data migration
```
Comment (vicmgs) [inline lib/console/alerts/alert.ex:37]: lmaoooo sounds good

File: lib/console/alerts/alerts.ex [hunk 2]
```diff
@@ -56,6 +60,21 @@ defmodule Console.Alerts do
...
+    case node_type do
+      "device" ->
+        Devices.get_device!(organization, node_id)
+      "label" ->
+        Labels.get_label!(organization, node_id)
+      "integration" ->
+        Channels.get_channel!(organization, node_id)
+    end
+
+    alert_node = Repo.insert(AlertNode.changeset(%AlertNode{}, %{ "alert_id" => alert.id, "node_id" => node_id, "node_type" => node_type }), on_conflict: :nothing)
```
Comment (vicmgs) [inline lib/console/alerts/alerts.ex:73]: doesn't Repo.insert return {:ok, _node} already?

File: lib/console/alerts/alert.ex (hunk 1 above)
Comment (ambuznego) [reply lib/console/alerts/alert.ex:37]: @vicmgs this was part of what was silently failing and causing a ruckus lol so this is to account for me creating an alert that is named "Alert for label #{label.name}` or "Alert for integrations prev. in label #{label.name}`

File: priv/repo/migrations/20210520192649_label_notificationss_to_alerts.exs [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+          end
+    end)
+
+    webhook_sql = """
+      SELECT w.key AS key, w.value AS value, w.label_id AS label_id, w.url AS url, w.notes AS notes, l.organization_id AS organization_id 
+        FROM label_notification_webhooks w LEFT JOIN labels l ON l.id = w.label_id
+          WHERE w.key = ANY($1)
+    """
+    results = Ecto.Adapters.SQL.query!(Console.Repo, webhook_sql, [events]).rows
+    IO.inspect results
```
Comment (vicmgs) [inline priv/repo/migrations/20210520192649_label_notificationss_to_alerts.exs:148]: log

File: lib/console/alerts/alerts.ex (hunk 2 above)
Comment (ambuznego) [reply lib/console/alerts/alerts.ex]: fixed!

File: priv/repo/migrations/20210520192649_label_notificationss_to_alerts.exs (hunk 3 above)
Comment (ambuznego) [reply priv/repo/migrations/20210520192649_label_notificationss_to_alerts.exs]: removed

**Review by vicmgs — APPROVED** [review]

**Pull request merged**